compiler/noirc_frontend/src/tests.rs

```diff
@@ -488,8 +488,9 @@ mod test {
             fn default(x: Field, y: NotAType) -> Field;
         }
         
-        fn main(x: Field, y: Field) {
-            assert(y == x);
+        fn main(x: Field, y: Field) -> pub bool {
```

Comment:
Why have these tests changed?

Reply:
Assertions now insert calls to functions that we have hidden through macros which they do not have access to in these tests. To get tests passing I originally just changed the tests, however, I can add usage of the noirc macros as the resolver expects them now anyway.

Reply:
Ok tests are back to as they were